DS13/code/modules/section31/datum_section31.dm

```diff
@@ -0,0 +1,376 @@
+		else
+			return forge_single_human_objective()
+
+/datum/antagonist/section31/proc/forge_single_human_objective() //Returns how many objectives are added
+	.=1
+	if(prob(50))
+		var/list/active_ais = active_ais()
+		if(active_ais.len && prob(100/GLOB.joined_player_list.len))
+			var/datum/objective/destroy/destroy_objective = new
+			destroy_objective.owner = owner
+			destroy_objective.find_target()
+			add_objective(destroy_objective)
+		else if(prob(30))
+			var/datum/objective/maroon/maroon_objective = new
+			maroon_objective.owner = owner
+			maroon_objective.find_target()
+			add_objective(maroon_objective)
+		else
+			var/datum/objective/assassinate/kill_objective = new
+			kill_objective.owner = owner
+			kill_objective.find_target()
+			add_objective(kill_objective)
+	else
+		if(prob(15) && !(locate(/datum/objective/download) in objectives) && !(owner.assigned_role in list("Research Director", "Scientist", "Roboticist")))
+			var/datum/objective/download/download_objective = new
+			download_objective.owner = owner
+			download_objective.gen_amount_goal()
+			add_objective(download_objective)
+		else
+			var/datum/objective/steal/steal_objective = new
+			steal_objective.owner = owner
+			steal_objective.find_target()
+			add_objective(steal_objective)
+
+/datum/antagonist/section31/proc/forge_single_AI_objective()
+	.=1
+	var/special_pick = rand(1,4)
+	switch(special_pick)
+		if(1)
+			var/datum/objective/block/block_objective = new
+			block_objective.owner = owner
+			add_objective(block_objective)
+		if(2)
+			var/datum/objective/purge/purge_objective = new
+			purge_objective.owner = owner
+			add_objective(purge_objective)
+		if(3)
+			var/datum/objective/robot_army/robot_objective = new
+			robot_objective.owner = owner
+			add_objective(robot_objective)
+		if(4) //Protect and strand a target
+			var/datum/objective/protect/yandere_one = new
+			yandere_one.owner = owner
+			add_objective(yandere_one)
+			yandere_one.find_target()
+			var/datum/objective/maroon/yandere_two = new
+			yandere_two.owner = owner
+			yandere_two.target = yandere_one.target
+			yandere_two.update_explanation_text() // normally called in find_target()
+			add_objective(yandere_two)
+			.=2
+
+/datum/antagonist/section31/greet()
+	SEND_SOUND(owner.current, 'DS13/sound/effects/section31/section31.ogg')
+	to_chat(owner.current, "<B><font size=3 color=red>You are the [owner.special_role].</font></B>")
+	to_chat(owner.current, "<span class='danger'>We are here under best interests of the higher echelons of Federation Command, we are the never blinking eye.</span>")
+	to_chat(owner.current, "<span class='danger'>No one of any authority is permitted to interfere with your objectives, complete them at all costs. Do not allow yourself to be captured, you are armed with a cyanide pill and various implants of explosives. Remove all nuisances and potential threats to yourself and the Federation by all means necessary.</span>")
```

Comment:
Text gives the impression of validating murderbone. Perhaps rephrase?

Reply:
Nope